src/DataFrameIndexingMsg.chpl

```diff
@@ -0,0 +1,204 @@
+    module DataFrameIndexingMsg
+    {
+    use ServerConfig;
+    use ServerErrorStrings;
+
+    use Reflection;
+    use ServerErrors;
+    use Logging;
+    use Message;
+    use SegmentedMsg;
+    use AryUtil;
+
+    use MultiTypeSymEntry;
+    use MultiTypeSymbolTable;
+
+    use CommAggregation;
```

Comment:
do we need this import if we're not using aggregation?

Reply:
Good point. This has been removed.